pkgs/development/tools/rust/cargo-lambda/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv, lib, rustPlatform, fetchFromGitHub, makeWrapper, cargo-watch }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "cargo-lambda";
+  version = "0.11.2";
+
+  src = fetchFromGitHub {
+    owner = pname;
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-IK4HVj8Y8Vz+mza8G9C+m5JRfNT3BWWdlbQQkJPu6RI=";
+  };
+
+  cargoSha256 = "sha256-oSqoSvv8IiChtduQQA31wItHcsnRBAQgOCrQN4sjcx8=";
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  postInstall = ''
+    wrapProgram $out/bin/cargo-lambda --prefix PATH : ${lib.makeBinPath [ cargo-watch ]}
```

Comment:
`cargo-watch` is a runtime dependency of  `cargo-lambda`. By ensuring that cargo-watch is available, it will not try and download `cargo-watch` automatically at runtime.